### PR TITLE
feat(DC): discover_events yaml

### PR DESCRIPTION
**Pull request by volokluev**
discover_events entity generated with the [entity conversion script](https://github.com/getsentry/snuba/tree/python_2_yaml_entity_converter)

Join relationships and subscription validators had to be updated manually

### CODE REVIEW

File: snuba/datasets/configuration/discover/entities/discover.yaml [hunk 1]
```diff
@@ -363,8 +363,8 @@ storages:
         - mapper: DefaultNoneSubscriptMapper
           args:
             subscript_names:
-              - measurements
               - span_op_breakdowns
+              - measurements
```
Comment (evanh) [inline snuba/datasets/configuration/discover/entities/discover.yaml:367]: Why this change?

**Review by evanh — APPROVED** [review]
One quick question.

File: snuba/datasets/configuration/discover/entities/discover.yaml (hunk 1 above)
Comment (volokluev) [reply snuba/datasets/configuration/discover/entities/discover.yaml:367]: It's an artifact of me changing the order in the discover.py file first and then seeing that it caused a difference in this file. More operations than necessary but no impact either way
